pyproject.toml

```diff
@@ -1,14 +1,15 @@
 [build-system]
-requires = ["setuptools>=67"]
+requires = ["setuptools>=67.5"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "archinstall"
-dynamic = ["version", "entry-points", "license"]
+dynamic = ["version", "entry-points"]
 description = "Arch Linux installer - guided, templates etc."
 authors = [
     {name = "Anton Hvornum", email = "anton@hvornum.se"},
 ]
+license = {text = "GPLv3"}
```

Comment:
Is it GPL-3.0 or GPL-3.0-or-later?

Reply:
Updating, it should be `GPL-3.0-only`. Couldn't remember what to search for but SPDX license identifiers was it, thanks! :D